pandas/core/indexes/base.py

```diff
@@ -1153,10 +1157,17 @@ def to_frame(self, index=True):
         0   Ant
         1  Bear
         2   Cow
+
```

Comment:
A bit of prose here explaining:

    To override the name of the resulting column, specify `name`: